wavefront/server/packages/flo_cloud/flo_cloud/azure/synapse.py

```
@@ -0,0 +1,264 @@
+                where_clause,
+                limit,
+                offset,
+                order_by,
+                group_by=group_by,
+            )
+        else:
+            query = (
+                f'SELECT {projection} FROM {base_table} AS a '
+                f'WHERE {where_clause} {group_by_clause} '
+                f'{order_by_clause} '
+                f'OFFSET {offset} ROWS FETCH NEXT {limit} ROWS ONLY'
```

Ruff (0.15.7)

[error] Possible SQL injection vector through string-based query construction

(S608)